selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/core/client/keycloak/OsioKeycloakTestAuthServiceClient.java

```diff
@@ -78,12 +78,16 @@ private KeycloakToken obtainActiveToken(String username, String password)
     HttpURLConnection conn;
     String formPostURL = loginAndGetFormPostURL();
 
+    // Need to disable followRedirects
     HttpURLConnection.setFollowRedirects(false);
 
     conn = fillFormAndCreateConnection(formPostURL, username, password);
 
     String tokenJsonString = followRedirects(conn);
 
+    // Re-enabling followRedirects
+    HttpURLConnection.setFollowRedirects(true);
```

Comment:
It would be more safe to use `conn.setInstanceFollowRedirects()` instead of `HttpURLConnection.setFollowRedirects()` here so that we don't change settings of `HttpURLConnection` globally.